src/style.css

```diff
@@ -1,3 +1,122 @@
+:root {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  --bg-color: #fff;
+  --a-color: #000;
+  --a-hover-color: #00a0e3;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url("Roboto/Roboto-Medium.ttf");
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+  color: black;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--bg-color);
+}
+
+.logo {
+  vertical-align: middle;
+  margin-top: 10px;
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  margin-top: 0;
+  padding-top: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  margin-top: 12.5px;
+  padding-top: 0;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  color: var(--a-color);
+  text-decoration: none;
+}
+
+.nav__link:hover::after {
+  content: "";
+  display: block;
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  bottom: 0;
+  border-radius: 8px;
+}
```

Comment:
You should only change the color on hover. No additional pseudo-elements needed!